internal/indexer/documentation.go

```diff
@@ -299,13 +299,9 @@ func (d *docsIndexer) indexFile(p *packages.Package, f *ast.File, isTestFile boo
 	var result fileDocs
 	result.pkgDocsMarkdown = godocToMarkdown(f.Doc.Text())
 
-	// Visit each AST node, collecting the bits we want to emit documentation for.
-	var (
-		firstErr error
-		visitor  ast.Visitor
-	)
-	visitor = visitorFunc(func(n ast.Node) (w ast.Visitor) {
-		switch node := n.(type) {
+	// Collect each top-level declaration.
```

Comment:
Oh this is probably much faster as well than walking the entire AST.

Reply:
yeah :D